### PR TITLE
Updated the command for GKE cluster creation in standard mode

### PR DESCRIPTION
**Pull request by Harsha6174**
Fixes # .

Change summary:
Updated the GKE standard cluster creation command to include the ones for monitoring and logging. 

The update parameters can be found in [this document](https://cloud.google.com/stackdriver/docs/solutions/gke/installing#gcloud).

### CODE REVIEW

File: v1.35.0 [hunk 1]
```diff
@@ -0,0 +1,1611 @@
+
```
Comment (Shabirmean) [inline v1.35.0:1]: Remove this additional file!

**Review by Shabirmean — CHANGES_REQUESTED** [review]
Thank you for the changes. There is a minor request for change!

**Comment by Harsha6174** [comment]
Thanks, @Shabirmean . I have removed the extra file (v1.35.0) that had cropped in.

**Review by Shabirmean — APPROVED** [review]
LGTM! Thank you for helping us keep the sample up to date

**Pull request merged**